source/guides/network/wifi.rst

```diff
@@ -0,0 +1,99 @@
+.. _wi-fi:
+
+Configure Wi-Fi with NetworkManager
```

Comment:
Suggest dropping "with NetworkManager" in the title and using "Wireless network" instead of/in addition to Wi-Fi in the title

Reply:
Done.